stable/prometheus/templates/alertmanager-statefulset.yaml

```diff
@@ -58,6 +58,12 @@ spec:
             - --config.file=/etc/config/alertmanager.yml
             - --storage.path={{ .Values.alertmanager.persistentVolume.mountPath }}
             - --cluster.advertise-address=$(POD_IP):6783
+          {{- if .Values.alertmanager.statefulSet.headless.enableMeshPeer }}
```

Comment:
please add this variables on Readme.md

Reply:
I did not introduce any new variables. These are already listed on the README.